src/libs/actions/Report.ts

```diff
@@ -4903,6 +4904,10 @@ function deleteAppReport(reportID: string | undefined) {
             return;
         }
 
+        if (!!report && !!reportTransactions && isDeleteAction(report, reportTransactions, [reportAction])) {
```

Comment:
```suggestion
        isDeletedAction(reportAction) {
```
Isn't the `isDeletedAction` from `ReportActionsUtils` enough here?